.github/workflows/codeql.yml

```diff
@@ -48,6 +56,29 @@ jobs:
         languages: ${{ matrix.language }}
         build-mode: ${{ matrix.build-mode }}
 
+    - name: Add NuGet sources
+      shell: pwsh
+      if: matrix.language == 'c-cpp'
+      run: |
+        .$(vcpkg fetch nuget) `
+          sources add `
+          -Source "${{ env.FEED_URL }}" `
+          -StorePasswordInClearText `
```

Comment:
What does this flag do?

Reply:
Stores password in clear text :P According to documentation,
> Be aware that encrypted passwords are only supported on Windows.

Binary caching was setup according to https://learn.microsoft.com/en-us/vcpkg/consume/binary-caching-github-packages?pivots=windows-runner, without double-checking whatever it does